utils/db/challenges.js

```diff
@@ -58,26 +67,38 @@ export async function setCurrentChallenge(teamId) {
 
 export async function getAllAssignedChallenges(userId) {
   const teamIds = await getCurrentValue(`/users/${userId}/teams`);
-  const data = await Promise.all(teamIds.map(async(teamId) => {
-    return {
-      challengeId: await getCurrentChallengeId(teamId),
-      team: await getTeam(teamId)
-    }
-  }));
-
-  const assignedChallenges = await Promise.all(data.map(async (assignedChallengeId) => {
-    let assignedChallenge = await getCurrentValue(`/assignedChallenges/${assignedChallengeId.challengeId}`);
-    let challengeDetails = await getCurrentValue(`/challenges/${assignedChallenge.challengeId}`);
-    if (assignedChallenge){
-      //Augment assignedChallenges with various team information
-      assignedChallenge["users"] = assignedChallengeId.team.users;
-      assignedChallenge["teamName"] = assignedChallengeId.team.name;
-
-      //Augment assignedChallenges with associated challenge details (like name of challenge, description etc)
-      assignedChallenge["challengeDetails"] = challengeDetails;
-      
-      return assignedChallenge;
-    }
-  }));
+  const data = await Promise.all(
+    teamIds.map(async teamId => {
+      return {
+        challengeId: await getCurrentChallengeId(teamId),
+        team: await getTeam(teamId)
+      };
+    })
+  );
+
+  const assignedChallenges = await Promise.all(
```

Comment:
These look like Prettier changes, but wanted to confirm

Reply:
Yeah, I also added the assignedChallengeId